src/main/java/org/mastodon/mamut/clustering/ClusterRootNodesController.java

```diff
@@ -104,10 +103,15 @@ public ClusterRootNodesController( final Model model, final BranchGraphSynchroni
 		this.synchronizer = synchronizer;
 	}
 
+	public Model getModel()
+	{
+		return model;
+	}
+
 	/**
 	 * Create a new tag set based on the current settings of the controller.
 	 */
-	public void createTagSet()
+	public void createTagSet( final boolean showDendrogram, final String tagSetName )
```

Comment:
I'm confused why all the other parameters have some kind of set method, but showDendrogram and tagSetName are passed directly? Feels inconsistent.

Reply:
Yes, indeed. This may be confusing in the future. I added both params as class variables